src/CISDSCResourceGeneration/functions/private/ConvertFrom-PrivilegeRightRawGPO.ps1

```diff
@@ -17,7 +17,8 @@ function ConvertFrom-PrivilegeRightRawGPO {
     process {
         $privilegeHash = @{
             'Policy' = $script:UserRights[$Policy]
-            'Identity' = ($Identity -split ',')
+            # The sort-object ensures consistent ordering of SIDs for the same settings accross resources to make comparisons easier.
+            'Identity' = ($Identity -split ',') | Sort-Object
```

Comment:
A minor thing, but do note that this does change the type of `Identity` from `string[]` to `object[]`. Implicit typing in PowerShell shouldn't be affected by this, but it's something to be aware of.

Reply:
Where are you seeing the object[]? This is what I see testing locally

![image](https://user-images.githubusercontent.com/28571284/106189725-47925000-616e-11eb-861e-e7a3cea5f056.png)

Reply:
Piping the object to `Get-Member` will iterate over each item in the array. Normally this is fine, because each item will be the same type. For array types you need to pass it as a parameter directly:
```
PS > $Identity = '*S-1-5-32-545,*S-1-5-32-544'
PS > $Identity_list = ($Identity -split ',') | Sort-Object
PS > Get-Member -InputObject $Identity_list

   TypeName: System.Object[]
```
vs.
```
PS > $Identity_list = ($Identity -split ',')
PS > Get-Member -InputObject $Identity_list

   TypeName: System.String[]
```

Reply:
Looks like passing the entire array in to inputobject vs piping will always have that result
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-member?view=powershell-7.1#example-6--get-members-for-an-array

Reply:
That's very interesting. I've forced it to remain a string[] in the latest commit

Reply:
I didn't expect you to make changes, but it looks like your most recent changes keeps the typing as it was before.